climate_scanner/sentiment_classifier/sentiment_classifier.py

```diff
@@ -328,12 +328,18 @@ def get_embeddings_mx(word_index):
 
 #############################################################################
 #
-# 	              SentimentClassifier class
+# 	              Interface class
 #
 #############################################################################
 
-class interface:
-    # interface class to use SentimentClassifier
+class sentiment_interface:
```

Comment:
The style guide on python suggests that classes follow camel casing: i.e. SentimentInterface rather than sentiment_interface.